torpaste.py

```diff
@@ -1,4 +1,4 @@
-#!bin/python
```

Comment:
This is true, however you don't want to run the OS' version of python, you want to run the local one, inside the `virtualenv`, which is always `bin/python` from what I know. Do you have an example where this is not the case? I am curious.